pkg/didcomm/crypto/jwe/authcrypt/authcrypt_test.go

```diff
@@ -434,80 +434,22 @@ func TestBadCreateCipher(t *testing.T) {
 	// https://github.com/hyperledger/aries-rfcs/issues/133#issuecomment-518922447
-	var senderPrivStr = "XZOBB1M7ikDoFR86rSgAuVt1ACJDMJ9alxHUsND6MBo"
-	senderPriv, err := base64.RawURLEncoding.DecodeString(senderPrivStr)
-	require.NoError(t, err)
-	var senderPubStr = "qdXzr6z28ck-RfTEiaBZmHOwH11ow-CBfLo97Qe31j4"
-	senderPub, err := base64.RawURLEncoding.DecodeString(senderPubStr)
-	require.NoError(t, err)
-	var recipientPrivStr = "kE3RDpviO_lVI3hdi6CKfT2BbuPph4WjC2DnkX7fBz4"
+	var recipientPrivStr = "95qt28Cq-lgqDXgDeN2zoVCMgdnDGjU4y_u4qzfSLg0"
 	recipientPriv, err := base64.RawURLEncoding.DecodeString(recipientPrivStr)
 	require.NoError(t, err)
-	var recipientPubStr = "800RcOPc9M8vFElpaHGkl-p9SpmY2Efm2MZW5tikf1c"
+	var recipientPubStr = "OBuFCIsVQhuIWZ5e3rB2Curr-2350js6n1SIdmZPRBQ"
 	recipientPub, err := base64.RawURLEncoding.DecodeString(recipientPubStr)
 	require.NoError(t, err)
-	var payloadStr = "SGVsbG8gV29ybGQh"
-	payload, err := base64.RawURLEncoding.DecodeString(payloadStr)
-	require.NoError(t, err)
 
 	// refJWE created by executing PHP test code at:
 	// https://github.com/gamringer/php-authcrypt/blob/master/examples/1-crypt.php
-	//nolint:lll
-	const refJWE = `{
-    "protected": "eyJ0eXAiOiJwcnMuaHlwZXJsZWRnZXIuYXJpZXMtYXV0aC1tZXNzYWdlIiwiYWxnIjoiRUNESC1TUytYQzIwUEtXIiwiZW5jIjoiWEMyMFAifQ",
-    "recipients": [
-        {
-            "encrypted_key": "-MXYFTdqmcmw11apipgtcr-E365Yvk6_4d9cVxRs89U",
-            "header": {
-                "apu": "cJn0DhdOCZAmuLCFuOM1R9v26aPVl-EMW5V_Y81zgkddrzw2WmAvdSbhrS0BHjAmRdsZ52fPYoveQZeQIIFqPw",
-                "iv": "s2rbhR-abOcLdJuZpvKLa_aLfhIqRyGL",
-                "tag": "vx1JrepSfW90QIbG7vRBWg",
-                "kid": "HNkELiimfV5S3VyMWVqCd6H77cE3FMhYgp2Gq4tvZRJn",
-                "oid": "KStwoVefDEH4gsdTQay0QyLkPMtXdJPJhMrO9hk-A0-cKE5sqBZxAIc4_iw7VOaSRLCMipZgYWD1epH1hQJbUMESQtuGUBCxVZCAJJYQNML7PtZz1wooCYyBfa4"
-            }
-        }
-    ],
-    "aad": "xuT9nzr1gf7k9IlS2936LFnUoDb-Tu1cBa8fhfUgxGk",
-    "iv": "nM0UDDCERek5syITAHwzDPGiEVErTtpo",
-    "tag": "gu1ZvF35-JYMd1JITD0qeg",
-    "ciphertext": "ntZwQokGaZhnQ8L2"
-}`
-
-	senderKp := jwecrypto.KeyPair{
-		Priv: senderPriv,
-		Pub:  senderPub,
-	}
+	const refJWE = `{"protected":"eyJ0eXAiOiJwcnMuaHlwZXJsZWRnZXIuYXJpZXMtYXV0aC1tZXNzYWdlIiwiYWxnIjoiRUNESC1TUytYQzIwUEtXIiwiZW5jIjoiWEMyMFAifQ","recipients":[{"encrypted_key":"-FqJHyCb52lPTU0GkjTLFHPv-bIjk7zV7j3yQmcUuaU","header":{"apu":"EbofDL0elLBv4pcoeyjo_tgx3Cd0SA6NqW5qWPAqU--7RTWKosW_7YzrA2KbbpEcAOCBv-GWoa9Hyrsuil_luQ","iv":"D5-segFndIlJRLAwSoFHKjtItV2DHSHr","tag":"wllnsUZ0LD1ktiu1dZoXWQ","kid":"4n2AqYZ9kzyTWWZJnS6wSuhkqzW9qUdQHpoWXSBG6KwR","oid":"Q_DA4thb6f5_IyZOTYQJjQHor336K3kb2erwfRqq6xmrHZTwv1OgwxikJErpDqJ6t-Cf27N2IWyvcV6Z0Z0a3322AZQmfV2nMXgyCWV42ad395jIE9zpPSvi6CE"}}],"aad":"jzdqnBug3dzkhEyWVzHA-JhCS7OHkVNxMp4DxVVxPAE","iv":"TEk7JtQZwsU0mBJeew9DH2npmxjr8_Mr","tag":"zFMdWklZaxCymIEPO_5Cuw","ciphertext":"WgRFQZJ_43UwtRQm"}` //nolint:lll
```

Comment:
This line could be formatted like it was before. Also is this change to ref JWE needed?

Reply:
older format had a linter error (lll) that I can't comment inside of a multi-line string

Reply:
You put the no lint comment in the line before the const.

Reply:
done